client/reducers/userReducer.js

```diff
@@ -0,0 +1,17 @@
+import defaultState from '../defaultState';
+import { SAVE_USER } from '../actionConstants/userConstant';
+
+const userReducer = (state = defaultState.loggedInUser, action) => {
+  switch (action.type) {
+    case SAVE_USER: {
+      const { user } = action;
+      return { ...state, loggedInUser: user };
```

Comment:
I believe you can just spread user here, since we do not have loggedInUser as a key in initialState for the reducer, and then, we can just keep all the user info in the top level object, no need to assign to another key in the reducer
`return { ...state, ...action.user };`